WordPress/src/main/java/org/wordpress/android/ui/qrcodeauth/compose/components/Subtitle.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.qrcodeauth.compose.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import org.wordpress.android.ui.compose.unit.FontSize
+import org.wordpress.android.ui.compose.unit.Margin
+
+@Composable
+fun Subtitle(text: String) {
```

Comment:
Same remarks as above component.

Reply:
Solved in [0b9b0b7](https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/0b9b0b7c2ed0902e034373d6aca9ce61f07563db)